src/common/constants.ts

```diff
@@ -1,4 +1,5 @@
 const MAP_API_KEY = process.env.GATSBY_MAP_API_KEY
+const GROWTHBOOK_CLIENT_KEY = process.env.GROWTHBOOK_CLIENT_KEY || 'sdk-0aHHaxKhwn0bP1HE'
```

Comment:
For gatsby, we'll need to rename the .env variable
```suggestion
const GROWTHBOOK_CLIENT_KEY = process.env.GATSBY_GROWTHBOOK_CLIENT_KEY
```

Reply:
Thanks @habib-deriv Fixed in 0408a3656ef082debc0dcc1c43072dd105c9d0b1